### ISSUE TITLE
Not Working in All Applications

### ISSUE THREAD

**Issue by Astra060** [moudey/Shell#454]
First of all, thank you for the amazing project. The Main problem I have is application which have their own Context Menu Hooks overwrite Shell. Is it possible to ignore these applications hooks and display Shell instead? I saw this behavior in Everything (The Application) and Directory Opus

**Comment by RubicBG** [comment]
https://github.com/moudey/Shell/issues/435
https://github.com/moudey/Shell/issues/412
https://github.com/moudey/Shell/issues/418

**Comment by oscarus** [comment]
Yeah, this would be nice to have. It's kinda jarring to have the context menu the way you want it just see the default context menu mess show up when using other programs.
I've tried modifying the 	`exclude.where = !process.is_explorer` , but that messes a bit with some programs' custom menus, like **everything** or **wiztree**.

The default menu also rears its ugly head when using a program's "open" or "save as" prompts.
